server/src/main/resources/db/changelog/changelog-testing.xml

```diff
@@ -2028,7 +2028,7 @@
     </changeSet>
     <changeSet author="awood" id="1413225753032-290">
         <comment>
-             Add the default quartz lock columns.
+             Add the default quartz lock columns. 
```

Comment:
Fixed. We may have an issue with our buildr changeset task, as I don't edit those changelog-* files directly.